pkg/client/metrics/prometheus/prometheus.go

```diff
@@ -27,26 +27,22 @@ import (
 	"github.com/prometheus/client_golang/prometheus"
 )
 
-const restClientSubsystem = "rest_client"
-
 var (
 	// requestLatency is a Prometheus Summary metric type partitioned by
 	// "verb" and "url" labels. It is used for the rest client latency metrics.
 	requestLatency = prometheus.NewSummaryVec(
```

Comment:
I considered it but as a default it might not be ideal:
* there are quite a few possible paths and histograms generate a fair amount of metrics – so unexpected blow-up
* we have to agree on a single bucketing pattern fitting latencies in all setups, which is not really possible – otherwise, people can just specify their own metrics in the first place
* I don't expect most clients to be part of a horizontally scaling service but rather controllers/operators/etc., which don't profit too much from histograms